httpMethods/cbHttp.js

```diff
@@ -22,18 +22,45 @@ class CbHttp {
         }
     }
 
-    getEstaciones = async(id) => {
+    getEstaciones = async (id) => {
         try {
-            const respuesta = id? await axios.get(this.url + "v2/entities/" + id, this.config) : await axios.get(this.url + "v2/entities/", this.config)
-            return {status : respuesta.status , 
-                mensaje : respuesta.data}
+            const respuesta = id ? await axios.get(this.url + "v2/entities/" + id, this.config) : await axios.get(this.url + "v2/entities/", this.config)
+            return {
+                status: respuesta.status,
+                mensaje: respuesta.data
+            }
         } catch (error) {
-            return {status : error.response.status , 
-                mensaje : error.response.data.name}
+            return {
+                status: error.response.status,
+                mensaje: error.response.data.name
+            }
+        }
+
+    }
+    suspenderEstacion = async (id) => {
+        let body = {
+            "enable": {
+                "value": false,
+                "type": "Boolean"
+            }
         }
-      
+        await axios.patch(this.url + "v2/entities/" + id + "/attrs/", body, this.config)
     }
+    getEstacionesPropias = async (user) => {
+        try {
+            let respuesta = await axios.get(this.url + "v2/entities/?q=ownerId=='urn:ngsi-ld:" + user + "'&options=keyValues&type=AirQualityObserved", this.config)
```

Comment:
Creo que los params no deberiamos hardcodearlos asi, sino pasarlos por parametros. Acá dejo un link de como hacerlo, pero basicamente es agregarlos a la config.
https://stackoverflow.com/questions/48261227/use-axios-get-with-params-and-config-together